extension/browserUtil.js

```diff
@@ -1,5 +1,7 @@
 /* globals buildSettings */
 import * as searching from "./searching.js";
+import * as content from "./background/content.js";
+const PAGE_LOAD_CHECKER_SCRIPT = "./content/pageLoadChecker.js";
```

Comment:
If something is only used once, it probably doesn't need to be a constant, you can just put the string in where it is used